test/lib/smart-survey-api.test.js

```diff
@@ -2,106 +2,60 @@
 
 const sinon = require('sinon');
 const chai = require('chai').use(require('sinon-chai'));
-chai.should();
 const proxyquire = require('proxyquire');
+chai.should();
+chai.use(require('chai-as-promised'));
```

Comment:
I'd put this stuff into a global config file for the unit tests so that you don't need to repeat it whenever you test with promises. It will also reduce any potential pain in context switching between different unit tests.

I think the HOF guys use this pattern. We also follow it in lev-web if you wanted to take a look.